### ISSUE TITLE
Concerningly orbfuscated

### ISSUE THREAD

**Issue by NNickey** [MR-NAYAN-404/nayan-media-downloader#2]
Code is obfuscated, slightly concerned if anything malicious may be included in the code.

**Comment by gensart-ai** [comment]
Yeah, I wonder why he put it on GitHub publicly but making it private.

**Comment by makarasty** [comment]
Download at your own risk, it seems to me that the developer is probably a novice...
according to the comits, this library uses `https:/snapsave.app/` ([06c7190](https://github.com/MR-NAYAN-404/nayan-media-downloader/commit/06c719077247b86bb31b412bcf1f135f6251da0a)) xd
